plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/configuration/BaseVersionedStorageFormat.java

```diff
@@ -27,13 +28,23 @@ public enum BaseVersionedStorageFormat implements VersionedStorageFormat {
    * make BlobDB more effective
    */
   FOREST_WITH_VARIABLES(DataStorageFormat.FOREST, 2),
+  /**
+   * Current Forest version, with receipts using compaction, in order to make Receipts use less disk
+   * space
+   */
+  FOREST_WITH_RECEIPT_COMPACTION(DataStorageFormat.FOREST, 3),
   /** Original Bonsai version, not used since replace by BONSAI_WITH_VARIABLES */
   BONSAI_ORIGINAL(DataStorageFormat.BONSAI, 1),
   /**
    * Current Bonsai version, with blockchain variables in a dedicated column family, in order to
    * make BlobDB more effective
    */
-  BONSAI_WITH_VARIABLES(DataStorageFormat.BONSAI, 2);
+  BONSAI_WITH_VARIABLES(DataStorageFormat.BONSAI, 2),
+  /**
+   * Current Bonsai version, with receipts using compaction, in order to make Receipts use less disk
+   * space
+   */
+  BONSAI_WITH_RECEIPT_COMPACTION(DataStorageFormat.BONSAI, 3);
```

Comment:
It's a one-way cumulation of features. You are right the name for `BONSAI_ORIGINAL_WITH_RECEIPT_COMPACTION` isn't quite right as it isn't Bonsai original + receipts, it also includes variables.

What about `BONSAI_VARIABLES_WITH_RECEIPT_COMPACTION`?

Reply:
On second thought I think will become a mess if we just keep accumulating the changes from every version into the name. Rather just keep it as is.

Reply:
Naming this was not easy for me, so I appreciate suggestions for improvements, perhaps we can just, name them like BONSAI_V3, and leverage the javadoc to specify that it is based on V2 plus the receipt compaction, so move the accumulation of the changes from the name to javadoc.

Note, that *_ORIGINAL are there only for reference, since it is not possible to use them anymore, so we could also remove them, or better document that are no more supported